text/0061-error-format-and-definitions.md

```diff
@@ -1045,9 +2131,9 @@ HTTP Code: `400 Bad Request`
 ```json
 {
     "message": "Task status `:status` is invalid. Available task statuses are: `:taskStatuses`.",
```

Comment:
Following my previous comment, we should clarify what we are talking about here :)